setup.cfg

```diff
@@ -2,7 +2,51 @@
 universal = 1
 
 [metadata]
+name = pycodestyle
+author = Johann C. Rocholl
+author_email = johann@rocholl.net
+maintainer = Ian Lee
+maintainer_email = IanLee1521@gmail.com
+license = Expat
 license_file = LICENSE
+description = Python style guide checker
+long_description = file: README.rst, CHANGES.txt
+keywords =
+    pycodestyle,
+    pep8,
+    PEP
+    8,
+    PEP-8,
+    PEP8
+url = https://pycodestyle.readthedocs.io/
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Libraries :: Python Modules
+
+[options]
+py_modules = pycodestyle
+zip_safe = False
+install_requires = setuptools>=42; wheel; read_version[toml] >= 0.3.0
```

Comment:
this is formatted incorrectly, also it's wrong -- pycodestyle has no runtime dependencies